packages/merge-on-green/src/merge-logic.ts

```diff
@@ -495,6 +495,24 @@ mergeOnGreen.commentOnPR = async function commentOnPR(
   }
 };
 
+mergeOnGreen.checkPRMerged = async function checkPRMerged(
+  owner: string,
+  repo: string,
+  pr: number,
+  github: GitHubAPI
+): Promise<boolean> {
+  try {
+    await github.pulls.checkIfMerged({
+      owner,
+      repo,
+      pull_number: pr,
+    });
+    return true;
+  } catch (err) {
```

Comment:
the fact that you're getting a `405` error seems odd to me, I would expect this to return a `true` or `false`, is the response of this function documented somewhere?